src/cameras/CameraRig.ts

```diff
@@ -281,8 +330,14 @@ export class CameraRig extends Node {
 					if (!cam) return
 
 					this.scrollFling!.y
-
-					untrack(() => (cam.position.z = this.scrollFling!.y))
+					const setScrollFactor = this.dynamicDolly
+					untrack(() => {
+						cam.position.z = this.scrollFling!.y
```

Comment:
The `untrack` could also be `untrack(() => cam.position).z =`.

This is one thing that needs to be fixed. It was based on an older pattern when things were all event based, and sub props (x/y/z) made the parent object emit an event (I basically ported it verbatim to Solid reactivity).

I think either we can remove reactivity for `.position` (and `.rotation`, etc) and have only the x/y/z be reactive, or have `.position` (etc) be Solid mutables (`return createMutable(this)`). I'm leaning on the first one.